opus/application/static_media/js/browse.js

```diff
@@ -624,12 +626,26 @@ var o_browse = {
 
     // find the first displayed observation index & id in the upper left corner
     updateSliderHandle: function() {
+        let tab = `#${opus.prefs.view}`;
+        let contentsView = o_browse.getScrollContainerClass();
```

Comment:
curiously, is the contenstView selector for the PS not the same as the infinite scroll container?

Reply:
It's the same as the infinite scroll container. (".op-gallery-view" and ".op-dataTable-view")